### DISCUSSION TITLE
Is it possible to create an island without naming it?

### DISCUSSION THREAD

**Discussion by Gr1ZzLy23** [BG-Software-LLC/SuperiorSkyblock2#1176] [Pending Feature Requests] [answered]
Hello, I'm wondering if it is possible to create an island without naming it and execute the command **/island create** directly without any **name** argument.

I also wonder if it is possible to turn off Safe Blocks completely, because the player can not teleport even on the newly created island.

**Comment by OmerBenGera** [comment, marked as answer]
You can turn off names in the config.
About safe blocks - if the player cannot teleport to his island, it means he either blocked all possible teleportation spots, broke all the blocks on his island, or if you use 1.12.2 - then there is an opened issue regarding this.
